test/CadlRanchProjects/arrays/item-types/Generated/Docs/UnknownValue.xml

```diff
@@ -9,20 +9,10 @@ var client = new ItemTypesClient().GetUnknownValueClient(<1.0.0>);
 
 Response response = await client.GetUnknownValueAsync();
 
-Console.WriteLine(response.ToString());
+JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
+Console.WriteLine(result[0].ToString());
```

Comment:
Can we provide more clear example to parse unknow return? maybe the raw response. Convert to string, sometime may not reasonable. It may not big problem, but it is better to provide more clear example, if we can.

Reply:
I did not go through the content of example generator yet.
I will take a look because this does not look correct.

Reply:
Here the code is totally expected here.
The API in this document is returning an array of unknown. We cannot do anything because it is unknown after we get `result[0]`. We do not know whether this is a primitive type or a complex object or another array.

The `ToString` invocation is irrelevant with the type here. In the doc here, we will ALWAYS call `ToString` because we are using `Console.Write` to show the result. I assume we could omit the `ToString` here because it is redundant, but I will not do this in this PR.

Reply:
We will handle in sample enhancement.